userGet.php

```diff
@@ -0,0 +1,57 @@
+<html>
+
+<head>
+<h1>User Profile:</h1>
+</head>
+
+<body>
+<div class=userGet>
+<?php
+
+// Enable error logging: 
+error_reporting(E_ALL ^ E_NOTICE);
+// mysqli connection via user-defined function
+include ('./my_connect.php');
+$mysqli = get_mysqli_conn();
+
+//echo '<p>' . $mysqli->host_info . '</p>';
+// gets value sent over search form
+$mid = $_GET['membership_ID']; 
+
+// (3) SQL statement to retireve user personal information for database and display them 
+$sql = "SELECT Member.membership_ID, Member.name, Member.date_of_birth, Member.subscription_type, Member.start_date, Member.end_date from Member\n"
+
+    . "WHERE Member.membership_ID like '$mid%'";
```

Comment:
SQL statement retrieving the user personal information works fine and displays it on a new page that follows the theme